src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -38,14 +38,14 @@ export default function IndexPatternFactory(Private, Notifier, config, kbnIndex,
   });
 
   const mapping = mappingSetup.expandShorthand({
-    title: 'string',
-    timeFieldName: 'string',
+    title: 'keyword',
```

Comment:
Are we sure we want these to be not-analyzed? We might want to use `text` in these cases.